scripts/globals/abilities/pets/crescent_fang.lua

```diff
@@ -20,9 +20,13 @@ function onPetAbility(target, pet, skill)
     local totaldamage = 0;
     local damage = AvatarPhysicalMove(pet,target,skill,numhits,accmod,dmgmod,0,TP_NO_EFFECT,1,2,3);
     totaldamage = AvatarFinalAdjustments(damage.dmg,pet,skill,target,MOBSKILL_PHYSICAL,MOBPARAM_PIERCE,numhits);
-    target:addStatusEffect(EFFECT_PARALYSIS, 22.5, 0, 90);
+    
+        if (damage.hitslanded > 0) then 
```

Comment:
now you over indented. your if was fine previous and all you had to do was put 4 spaces on the thing *inside* the if/end block.